circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   node:
-    version: 5.1.0
+    version: 6.2.1
```

Comment:
Version 6.1 is preinstalled on the "Trusty" image, so your tests run faster if you stick with that version. You have to change the image in the Circle settings.